File: java/resistor-color-duo/ResistorColorDuo.java
class ResistorColorDuo {
    private enum Colors {
        black, brown, red, orange, yellow, green, blue, violet, grey, white
    }

    int value(String[] colors) {
        int resistor_1 = Colors.valueOf(colors[0]).ordinal();
        int resistor_2 = Colors.valueOf(colors[1]).ordinal();

        return resistor_1 * 10 + resistor_2;
    }
}

File: python/resistor-color-duo/resistor_color_duo.py
_COLORS = ['black', 'brown', 'red', 'orange', 'yellow',
           'green', 'blue', 'violet', 'grey', 'white']

def value(colors: [str, str]) -> str:
    return _COLORS.index(colors[0]) * 10 + _COLORS.index(colors[1])
